### PR TITLE
Update how-to-access-query-string-parameters.md

### PR DESCRIPTION
**Pull request by Yash-Handa**
Hey @chowdhurian and @fhemberger,
In accordance with issue #1977, I have updated the Article **how-to-access-query-string-parameters** with the following changes:

- Added a note on how to execute the program.
- Added some detail about other key values provided by `url.parse()`.
- some other miscellaneous updates and refactors.

If there are any suggestion/enhancement/tips please feel free to tell me, I would love to incorporate those changes in this Article 😄

### CODE REVIEW

File: locale/en/knowledge/HTTP/clients/how-to-access-query-string-parameters.md [hunk 1]
```diff
@@ -9,18 +9,27 @@ layout: knowledge-post.hbs
...
+> To test this code run `node app.js` (app.js is name of the file) on the terminal and then go to your browser and type `http://localhost:8080/app.js?foo=bad&baz=foo` on the URL bar
 
+The key part of this whole script is this line: `const queryObject = url.parse(req.url,true).query;`. Let's take a look at things from the inside-out.  First off, `req.url` will look like `/app.js?foo=bad&baz=foo`. This is the part that is in the URL bar of the browser. Next, it gets passed to `url.parse` which parses out the various elements of the URL (NOTE: the second paramater is a boolean stating whether the method should parse the query string, so we set it to true). Finally, we access the `.query` property, which returns us a nice, friendly JavaScript object with our query string data. 
 
+The `url.parser()` method returns an object which have many key value pairs one of which is the `query` object. Some other handy information returned by the method include `host`, `pathname`, `search` keys.
```
Comment (jonchurch) [inline locale/en/knowledge/HTTP/clients/how-to-access-query-string-parameters.md]: This should be `url.parse` instead of `parser`

**Comment by jonchurch** [comment]
I was working on this a moment ago hehe, I had added a section mentioning the builtin `querystring` module, explaining the differences between parsing with `url` and `querystring`.

> ## Parsing with `querystring`
> Another way to access query string parameters is parsing them using the `querystring` builtin Node module.
> 
> This method, however, must be passed just a querystring portion of a url. Passing it a whole url, like we did in the `url.parse` example, won't properly parse the querystrings.
> 
> ```js
> const querystring = require('querystring')
> const qs = "code=string&key=12&id=false"
> const url = "http://example.com/index.html?code=string&key=12&id=false"
> 
> console.log(querystring.parse(qs))
> // > { code: 'string', key: '12', id: 'false' }
> 
> console.log(querystring.parse(url))
> // > { 'http://example.com/index.html?code': 'string', key: '12', id: 'false' }
> ```


**I also edited the initial example, since the HTTP server wasn't specific to the example.**

> ## Parsing with `url`
> In Node.js, functionality to aid in the accessing of URL query string parameters is built into the standard library. The built-in `url.parse` method takes care of most of the heavy lifting for us.
> 
> You can parse a url string into an object, including the querystrings, by passing a url to the built in `url` module's `parse` method.
> 
> To parse the querystrings themselves into an object, you must pass `true` as the second parameter to `url.parse`. Without passing true, the query value of the url object will just a single string.
> 
> Once parsed, we can access the query string parameters from the `query` property on the returned url object.
> 
> ```js
> const url = require("url");
> const urlString = "http://example.com/index.html?code=string&key=12&id=false";
> 
> const urlObject = url.parse(urlString, true);
> const queryObject = urlObject.query;
> 
> console.log(queryObject);
> // > { code: 'string', key: '12', id: 'false' }
> 
> ```

**Comment by Yash-Handa** [comment]
Thanks, @jonchurch, for such thoughtful content😄
I have added the `querystring` part to the Article. I would suggest keeping the `HTTP server` creation part in the Article because it serves as a complete and practical example.

If you further like to update other Articles in the [Knowlagebase](https://github.com/nodejs/nodejs.org/tree/master/locale/en/knowledge) I would suggest:
1. Select 4-5 articles to start with (or any number of articles you comfortable with).
2. Put the list of selected Articles on issue #1977 with some summary like in [here](https://github.com/nodejs/nodejs.org/issues/1977#issuecomment-460779022).
3. And get started 🎉🎉

We would love to have your contributions.🙂

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
@jonchurch：Please have a review again if possible, there're changes according to your suggestions :)

**Review by jonchurch — APPROVED** [review]

**Review by keywordnew — APPROVED** [review]
LGTM

**Pull request merged**